pbs/usersync.go

```diff
@@ -81,7 +81,7 @@ func (deps *UserSyncDeps) OptOut(w http.ResponseWriter, r *http.Request, _ httpr
 	pc.SetOptOut(optout != "")
 
 	// Write Cookie
-	encodedCookie, err := pc.PrepareCookieForWrite(deps.HostCookieConfig, encoder)
+	encodedCookie, err := pc.PrepareCookieForWrite(deps.HostCookieConfig, encoder, &usersync.OldestEjector{}) // TODO: Getting Non Priority Keys Here
```

Comment:
At this stage of the PR, trimming the cookie down to size is broken, and the priority ejector is not referenced in the production code?

Reply:
I'll reach out to you Monday to talk more about this part of the PR!